report-pr-errors

```diff
@@ -238,25 +238,25 @@ class Logs(object):
             except Exception as e:
                 print("Failed to upload %s to %s in bucket %s.%s" % (src, dst, bucket_name, server))
 
+
+def get_pending_namespace(args):
+    '''Return a Namespace to set status to pending with an appropriate message.'''
+    mesos_id = 'build/' + '/'.join(
+        re.match("^thermos-([a-z]*)-([a-z]*)-([a-z0-9_-]*)-([0-9]*)(?:-[0-9a-f]*){5}$",
+                 os.environ["MESOS_EXECUTOR_ID"]).groups())
```

Comment:
I would fallback to hostname if MESOS_EXECUTOR_ID is not there. Not all the checkers run mesos.

Reply:
Done!